pho/src/components/PopularSpace/SpaceCell/SpaceCell.tsx

```diff
@@ -17,14 +17,23 @@ import noOnlUsersIcon from '@/asset/noOnlUsersIcon.png';
 import SpaceModal from '@/components/WorldMap/SpaceModal';
 import { GetSpaceDetailResponse } from '@/lib/apis/space';
 
-const SpaceCell: React.FC<{ space: GetSpaceDetailResponse }> = ({ space }) => {
+import './SpaceCell.css';
+
+interface SpaceCellProps {
+  isDrawerOpen: boolean;
+  space: GetSpaceDetailResponse;
+}
+
+const SpaceCell: React.FC<SpaceCellProps> = ({ isDrawerOpen, space }) => {
   const { isOpen, onOpen, onClose } = useDisclosure();
   const URL = process.env.REACT_APP_BACKEND_URL ?? '';
   const [spaceInfo, setSpaceInfo] = useState<GetSpaceDetailResponse>();
 
   useEffect(() => {
     const fetchSpaceInfo = async () => {
       const spaceInfo = (await axios.get(`${URL}/api/spaces/${space.id}`)).data;
+      console.log(spaceInfo);
```

Comment:
I think we should remove all unneccessary log to advoid confusing while debug

Reply:
Resolved!